frontend/src/components/WofSmallError.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+<div class="small-error">
+    <div class="small-error__contener">
+        <div class="small-error__icon">
+            <wof-icon @mouseover="hover = true" @mouseleave="hover = false" icon="wof-warning-circle-fill" :size="this.size"/>
+        </div>
+        <div class="small-error__wrap-text">
+            <div class="small-error__content" v-if="hover">
+                <slot></slot>
+            </div>
+        </div>
+    </div>
+</div>
+</template>
+
+<script>
+export default {
+  name: "WofSmallError",
+
+  props: {
+      size: Number
+  },
+  data() {
+    return {
+      hover: false
+    };
+  }
+}
+</script>
+
+<style lang="less">
+
+.small-error{
+    position: absolute;
+    
+    .small-error__contener{
+        display: flex;
+        position: absolute;
+        min-width: 0;
+
+        .small-error__icon{
+            color: #E13E3E;
+            margin-right: 0.5em;
+            order: 1;
+            margin: 0.5em;   
```

Comment:
margin-right and margin defined at the same time? Is there a reason you didn't merge that property?